mg.c

```diff
@@ -1883,6 +1885,79 @@ Perl_magic_setsigall(pTHX_ SV* sv, MAGIC* mg)
     return 0;
 }
 
+int
+Perl_magic_gethook(pTHX_ SV *sv, MAGIC *mg)
+{
+    PERL_ARGS_ASSERT_MAGIC_GETHOOK;
+    PERL_UNUSED_ARG(sv);
+    PERL_UNUSED_ARG(mg);
+    return 0;
+}
```

Comment:
If you're not doing anything on get you can leave it out (see the hints hooks for example.)

Reply:
Ack. Ill remove this. I was thinking maybe it would come in useful and so it might as well be there for when it was needed, but i guess if someone wants it they can add it themselves.

Reply:
Anyway, done now. Thanks for the feedback!